packages/@sanity/types/src/schema/definition/schemaDefinition.ts

```diff
@@ -0,0 +1,131 @@
+import {PreviewConfig} from '../preview'
+import {InitialValueProperty, SchemaValidationValue} from '../types'
+import {
+  ArrayDefinition,
+  BlockDefinition,
+  BooleanDefinition,
+  CrossDatasetReferenceDefinition,
+  DateDefinition,
+  DatetimeDefinition,
+  DocumentDefinition,
+  FileDefinition,
+  GeopointDefinition,
+  ImageDefinition,
+  NumberDefinition,
+  ObjectDefinition,
+  ReferenceDefinition,
+  SlugDefinition,
+  SpanDefinition,
+  StringDefinition,
+  TextDefinition,
+  UrlDefinition,
+} from './type'
+import {BaseDefinitionOptions} from './type/common'
+
+/**
+ * This type can be extended being a interface and not a type, via declaration merging
+ */
+export interface IntrinsicTypeDefinition {
```

Comment:
To follow the naming convention of the other types, should this be named `IntrinsicDefinition`?

Reply:
Agreed. But we should add a release note on this, cause if anyone is extending this type to add their own intrinsics, it will just silently break.

Reply:
(I'm guessing very few have done that.)